index-rna.md

```diff
@@ -2,4 +2,192 @@
+
+**Want to learn more about RNA analyses? Take one of our guided tour or check our tutorials**
+
+Lesson | Slides | Hands-on | Tour
+--- | --- | --- | ---
+Introduction to Transcriptomics | [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/transcriptomics/slides/introduction.html){:target="_blank"} |  |  |
+Reference-based RNA-Seq data analysis | | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html){:target="_blank"}  | [<i class="fa fa-magic" aria-hidden="true"></i>](https://rna.usegalaxy.eu/tours/rnateam.rnaseq){:target="_blank"}
+Visualization of RNA-Seq results with CummeRbund | [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/slides.html){:target="_blank"} | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.html){:target="_blank"} |
+ViennaRNA Introduction | | | [<i class="fa fa-magic" aria-hidden="true"></i>](https://rna.usegalaxy.eu/tours/rnateam.viennarna){:target="_blank"}
+
+# Content
+{:.no_toc}
+
+1. TOC
+{:toc}
+
+# About the RNA Galaxy workbench
+
+The RNA Galaxy workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the Galaxy framework, which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.
+
+The current implementation comprises more than 50 bioinformatics tools dedicated to different research areas of RNA biology, including RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling, RNA-Seq analysis, and RNA target prediction.
+
+The workbench is developed by the RNA Bioinformatics Center (RBC). This center is one of the eight service units of the [German Network for Bioinformatics Infrastructure](http://www.denbi.de), running the German [ELIXIR Node](https://www.elixir-europe.org/).
+
+# Available tools
+
+In this section we list all tools that have been integrated in the RNA workbench. The list is likely to grow as soon as further tools and workflows are contributed. To ease readability, we divided them into categories.
+
+## RNA structure prediction and analysis
+
+Tool | Description | Reference
+--- | --- | ---
+[antaRNA](http://www.bioinf.uni-freiburg.de/Software/antaRNA/) | Possibility of inverse RNA structure folding and a specification of a GC value constraint | [Kleinkauf et al. 2015](https://doi.org/10.1093/bioinformatics/btv319)
```

Comment:
Just looked over this and these are all external links? I found this a bit strange after "Tools available on our server", and then being redirected externally. And some of them on http like our site will not load in an iframe in the usegalaxy.eu home page. I hate the "automatically open in new tab", but, yeah.

Reply:
I will change them to link to tool on the server